include/session/config/groups/members.hpp

```diff
@@ -489,6 +520,33 @@ class Members : public ConfigBase {
             return copy;
         }
     };
+
+    protected:
+        oxenc::bt_dict extra_data() const override {
+            oxenc::bt_dict result;
+            if (pending_send_ids && !pending_send_ids->empty()) {
+                oxenc::bt_list ids = {};
+                for (const auto& sid : *pending_send_ids)
+                    ids.emplace_back(sid);
+
+                result["pending_send_ids"] = ids;
+            }
+            
+            return result;
+        }
```

Comment:
With the producer change that I just pushed, and the change away from optional, this whole thing ought to be able to become just:
```suggestion
        void extra_data(oxenc::bt_dict_producer extra) const override {
            extra.append(pending_send_ids);
        }
```

(You might need to update oxen-encoding to the latest `dev` branch for that to work, and for that you might need to cherry-pick oxenc compat fix da5a2f3 from my multipart config PR).